protos/flow.proto

```diff
@@ -71,6 +71,8 @@ message FlowConnectionConfigs {
   string synced_at_col_name = 19;
```

Comment:
`TableSchema` uses string regardless of whether it is QValue/PGtype.

Use an enum or similar to help distinguish